packit/config/config.py

```diff
@@ -99,6 +100,10 @@ def __init__(
         self.command_handler_image_reference: str = command_handler_image_reference
         # do I really need to explain this?
         self.command_handler_k8s_namespace: str = command_handler_k8s_namespace
+        # To be able to mount other volumes (like repository cache) to the sandcastle pod
+        self.command_handler_pvc_volume_specs: List[Dict[str, str]] = (
+            command_handler_pvc_volume_specs or []
+        )
```

Comment:
@jpopelka What do you think about this approach? I don't like that we need to template `packit-service.yaml`. A counter-proposal can be to set this via environment-variable that will allow us to avoid templating but does not look nice to me.

Reply:
would be nice to request specific key names there: volume name and path

Reply:
would be nice to request specific key names there: volume name and path

Reply:
As we've discussed in person some time ago, it's not needed to be so strict for now -- we have the config on our control and the approach with repository-cache is not fully clear so we might need to prototype more quickly.